server/asset/eth/eth.go

```diff
@@ -173,8 +173,8 @@ func (eth *Backend) InitTxSizeBase() uint32 {
 }
 
 // FeeRate returns the current optimal fee rate in gwei / gas.
-func (eth *Backend) FeeRate() (uint64, error) {
-	bigGP, err := eth.node.suggestGasPrice(eth.rpcCtx)
+func (eth *Backend) FeeRate(ctx context.Context) (uint64, error) {
+	bigGP, err := eth.node.suggestGasPrice(ctx)
```

Comment:
With ETH's IPC RPCs, I'm less sure about this, but if the rule is to shutdown consumers (and cancel these passed contexts) before shutting down the `Backend`, we may not need the `rpcCtx` if we give more methods a `Context` arg.